maps/kibana/layers/10mriverslakecenterl/1b70dba2-10m-rivers-lake-centerlines.dbf

```diff
@@ -0,0 +1 @@
+/Users/cjcenizal/Documents/MapBox/cache/1b70dba2-10m-rivers-lake-centerlines/1b70dba2-10m-rivers-lake-centerlines.dbf
```

Comment:
I just experimented with deleting all of the directories inside of /layers, and it didn't affect TileMill. I'm guessing that this is because these layers ship with TileMill, so it's able to find them in the cache, even if they're missing from the project's /layers directory.